site/src/pages/WorkspacesPage/WorkspacesTable.tsx

```diff
@@ -643,6 +643,21 @@ const WorkspaceActionsCell: FC<WorkspaceActionsCellProps> = ({
 				}}
 				type="delete"
 			/>
+
+			{/* Cancel workspace build confirmation dialog */}
+			<ConfirmDialog
+				open={isCancelConfirmOpen}
+				title="Cancel workspace build"
+				description={`Are you sure you want to cancel the build for workspace "${workspace.name}"? This will stop the current build process.`}
```

Comment:
Would be nice if this could differ based on whether it thinks it's going to cancel a pending or running build.

e.g. `This will remove the current build from the build queue.`

Reply:
Same with the Dialog on the other page too, maybe it should be made into a component?